lib/solitude/tests.py

```diff
@@ -635,6 +640,12 @@ def test_callback_validation_failure(self):
         is_valid = self.provider.is_callback_token_valid({'foo': 'bar'})
         eq_(is_valid, False)
 
+    @raises(ValueError)
+    def test_no_usd(self):
+        self.provider.provider.create_transaction(
+            None, None, None, None, None, None, None,
+            {}, None, None, None, None)
```

Comment:
can't you just use ``self.configure(prices=None)`` or something along those lines?